Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -39,27 +39,27 @@ sys.path.append(SCRIPTS)
 from webkitpy.common.checkout.diff_parser import DiffParser
 from webkitbugspy import radar
 
-
-BRANCH = subprocess.check_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD'], **ENCODING_KWARGS).strip()
-for name, variable in dict(
-    title='COMMIT_MESSAGE_TITLE',
-    bug='COMMIT_MESSAGE_BUG',
-    cherry_picked='GIT_WEBKIT_CHERRY_PICKED',
-).items():
-    if variable in os.environ:
-        continue
-    try:
-        value = subprocess.check_output(
-            ['git', 'config', '--get-all', 'branch.{}.{}'.format(BRANCH, name)],
-            stderr=subprocess.PIPE,
-            **ENCODING_KWARGS
-        ).strip()
-        if value:
-            os.environ[variable] = value
-    except subprocess.CalledProcessError as error:
-        # `1` means that key did not exist, which is valid.
-        if error.returncode != 1:
-            sys.stderr.write(error.stderr)
+def set_env_variables():
```

Comment:
Nit: I think this should be `set_env_variables_from_config`. Reason for the verbose name is that, as you found, this is not intuitive behavior.